src/Math-Tests-Random/PMExponentialGeneratorTest.class.st

```diff
@@ -7,6 +7,14 @@ Class {
 	#category : #'Math-Tests-Random'
 }
 
+{ #category : #tests }
+PMExponentialGeneratorTest >> generateASampleOfSize: sampleSize UsingGenerator: gen [
```

Comment:
That's the same method as in `PMPoissonGeneratorTest`. Can we have this only once? Maybe in a common superclass of all random-number tests?

Reply:
I've actually proposed above moving the duplicate code to a helper class for now. I did think about moving the code to a super class but I tend to prefer composition to inheritance.